examples/resources/grafana_cloud_private_datasource_connect/resource.tf

```diff
@@ -0,0 +1,17 @@
+data "grafana_cloud_stack" "current" {
+  stackID = "<your stack ID>"
+}
+
+resource "grafana_cloud_private_datasource_connect" "test" {
+  region       = "us"
+  name         = "my-pdc"
+  display_name = "My PDC"
+  identifier   = data.grafana_cloud_stack.current.stackID
+}
+
+resource "grafana_cloud_private_datasource_connect_token" "test" {
+  pdc_network_id = grafana_cloud_private_datasource_connect.test.network_id
+  region         = "us"
```

Comment:
could we pass the whole network resource as input? Then we could get the region from the network schema, and we wouldn't need to repeat the `region` input in the token schema. This would feel more natural to me, since a token MUST be in the same region as its Access Policy

Reply:
I don't think this is straightforward. I've updated the example to use the region from the previously created network.

Reply:
Fair enough. I'll leave this comment open in case someone with some TF experience has input.

Reply:
I also see that this follows the same practice as the existing access policy token terraform, the consistency is more important 👍